Dockerfile.dev

```diff
@@ -6,14 +6,14 @@ RUN npm install -g pnpm@7.5.2
 
```

Comment:
@saleor/checkout & @magul 

How about we make 1-Dockerfile per package? 
We eventually need one for Checkout - do we want to have 2 services in 1 Dockerfile (doesn't seem right).

Maybe we could have 
`/Dockerfile.storefront` &
`/Dockerfile.checkout`

or 

`/apps/storefront/Dockerfile` &
`apps/checkout/Dockerfile`

It's on docker-compose to decide what services to include

If we decide now, we will avoid further braking changes in platform

What do you think?

Reply:
I was thinking about delivering `Dockerfile`, that will actually run storefront only, but as it relies on multiple of other npm workspaces, ultimate I've just:
```
COPY . .
```
assuming that could be considered good first approximation of what we want to achieve here, but I'm open for suggestions, how to actually deliver so fine-grained docker images without extensive overhead to be maintained.

Reply:
I think it's OK to copy everything (can be optimized later). My idea is to start only single service. 

Also, copy & install can be cached on docker layer AFAIK so only mainly CMD part will differ, right?